### PR TITLE
Refactor to es6 syntax

### PR DESCRIPTION
**Pull request by Edward-Kabue**
## Re-created  Awesome books app using 

- Use ES6 syntax.
- Divided my code into modules.
- Refactored my methods and functions to arrow functions.
- Use let and const in a correct way.

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Edward-Kabue, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] It is good to have a live link to the project, and It should be displayed in the README file.
- [ ] In the `Getting started ` section of the README file, The process to run the file locally is wrong, Please provide the necessary code to run the project locally.For ex:-
```
git clone https://github.com/Edward-Kabue/awesome-es6.git
cd awesome-es6
```
& Then right-click on `index.html` and select open with `live server`

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  if (title.value === '' || author.value === '') {
+    const error = document.getElementById('error');
+    error.textContent = 'Title and Author fields must be filled out';
+    error.style.color = 'red';
+    e.preventDefault();
+  } else {
+    const addBk = new Books(title.value, author.value);
+    addBk.addBook();
+  }
+});
```
Comment (medaminedev66) [inline index.js:33]: - [ ] When I add the second book and I click on `List`, I don't find the second book layout added to the list. Kindly fix this issue.

- After clicking on `Add book`:
image

- After clicking on `List`:
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Edward-Kabue, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Edward-Kabue, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**